examples/Replay

```diff
@@ -0,0 +1,46 @@
+#include <Servo.h>
+#include <EEPROM.h>
+
+Servo myServo;        // create servo object to control a servo
+
+bool record = false;  // boolean to determine if recording is taking plce
+int potpin = 0;       // analog pin used to connect the potentiometer
+int val;              // variable to read the value from the analog pin
+int read;             // variable to read the data from memory
+
+void setup() {
+  pinMode(13, OUTPUT); // button on pin 2
+  myServo.attach(9);   // attaches the servo on pin 9 to the servo object
```

Comment:
```suggestion
  myServo.attach(servoPin);   // attaches the servo pin to the Servo object
```
- You used a variable to define the potentiometer's pin, so to be consistent use a variable to define the servo's pin as well. I'll leave it to you to add the declaration of servoPin.
- Now that we have a nice descriptive variable name as the argument instead of a magic number, there is no need to specify the pin number in the comment.
- Use correct class name.